Nef_S2/include/CGAL/Nef_S2/SM_point_locator.h

```diff
@@ -257,7 +257,7 @@ class SM_point_locator : public SM_decorator {
     Unique_hash_map<SHalfedge_handle,bool> visited(false);
```

Comment:
I am wondering if this change: https://github.com/CGAL/cgal/pull/5486/commits/f72a006cd5cb328a740dd5e4bb920d86a9dffd77 makes a noticeable difference in your benchmarks.